package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/gwer/oncogene#readme",
   "devDependencies": {
     "babel-cli": "^6.23.0",
+    "babel-preset-babili": "0.0.12",
```

Comment:
I clone repo, then run npm install && npm run build, it generates empty js file. Then I update nodejs to latest version, this not help me. Then I add babili dependency and all works fine.